### DISCUSSION TITLE
Question: What distro and what desktop recommends to WSLG

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/wslg#223] [Q&A]
Hi friends!

I have 3 questions: 
 * 1 -What of the availibles distro you recommend to use with wslg (ubuntu, kali, debian, etc)
 *  2- What desktop Environment kind work better (gnome apps, kde, etc)
 *  3- Wich protocol work better x11 or wayland?

Regards and congratulations for the great work!

**Comment by Maks1116** [comment]
Are you sure that this needs bug label?

**Comment by ghost** [comment]
Hi, I try to remove the "bug" label, but with my permissions i cant or i don't find where. I you or some other can change it, please remove.

**Comment by onomatopellan** [comment]
The Kali-linux distro after installing the kali-win-kex package is the best looking distro for WSLg. The file explorer Thunar looks great in dark mode.

The [Ubuntu on Windows Community Preview](https://ubuntu.com/blog/announcing-ubuntu-on-windows-community-preview-wsl-2) distro is the most updated and is the best to try new things with WSLg since it has mesa GPU acceleration enabled by default.

Wayland looks better and "feels" faster. Try opening latest nightly from firefox in Wayland (`MOZ_ENABLE_WAYLAND=1 ./firefox`) and you will notice the difference.
But it's still a work in progress and X11 is way more mature.

**Comment by ghost** [comment]
Thanks for sharing this @onomatopellan!!!

**Comment by spronovo** [comment]
converted to a discussion.

The beauty of WSL is that you can use whichever Linux distribution you prefer, they should all work equally well... so try a few until you find your favorite, there is no right and wrong answer here :-).
